src/Components/Body/ChatArea/ChatArea.tsx

```diff
@@ -0,0 +1,60 @@
+import React, { useContext } from "react";
+import { ChatBody } from "./ChatBody/ChatBody";
+import { MessageBox } from "./MessageBox/MessageBox";
+import { ChatHeader } from "./ChatHeader/ChatHeader";
+import { useQuery } from "../../../Hooks/useQuery";
+import { UserContext } from "../../../App";
+import { NewMemberForm } from "./NewMemberForm/NewMemberForm";
+import { useModal } from "../../../Hooks/useModal";
+import "./ChatArea.css";
+
+interface chatAreaProps {
```

Comment:
1. Prefer `type` over `interfaces`. Just a convention
2. Types should always be capitalized. It should be `ChatAreaProps`